config/import-control.xml

```diff
@@ -4,7 +4,8 @@
     "https://checkstyle.org/dtds/import_control_1_4.dtd">
 
 <import-control pkg="com.puppycrawl.tools.checkstyle">
-
+  <!-- See https://github.com/checkstyle/checkstyle/issues/16903 -->
```

Comment:
Please remove comment.
If you want to bind update to this issue, put it as prefix in commit message

Reply:
yes, its well documented by git and the very nice project structure ensured.